roller.go

```diff
@@ -26,12 +25,12 @@ func adjust(asgList []string, ec2Svc ec2iface.EC2API, asgSvc autoscalingiface.Au
 	// get information on all of the ec2 instances
 	instances := make([]*autoscaling.Instance, 0)
 	for _, asg := range asgs {
-		oldI, newI, err := groupInstances(asg, ec2Svc)
+		oldInstances, newInstances, err := groupInstances(asg, ec2Svc)
```

Comment:
You really don't like `oldI` and `newI`, do you? :-) 

That is fine. Sometimes my roots come out in terseness. I am good with something more explicit.